### PR TITLE
add raw datasheets for danews2.0 memo nordjylland_news scrape_hovedst…

### PR DESCRIPTION
**Pull request by TTTTao725**
…aden swedish_gigaword

### CODE REVIEW

**Comment by TTTTao725** [comment]
Not sure what licenses they have yet.

File: docs/datasheets/danews2.0.md [hunk 1]
```diff
@@ -1,2 +1,55 @@
...
+## Data Fields
+
+- **id**: source-specific identifier.
+- **text**: textual content of the document.
+- **source**: source of the data.
+- **added**: timestamp ai2 acquired this data.
+- **created**": timestamp when original document was created (best-guess if not available)
+- **metadata**: source-specific metadata.
+
+## Lisence Information
```
Comment (KennethEnevoldsen) [inline docs/datasheets/danews2.0.md:47]: Should be License Information

File: docs/datasheets/danews2.0.md [hunk 2]
```diff
@@ -1,2 +1,55 @@
...
+  - text-generation
+  - fill-mask
+task_ids:
+  - language-modeling
+---
+# Dataset Card for Danews2.0
+## Dataset Description
+- **Number of records:** 26254962
+- **Languages:** Danish
+## Dataset Sturcture
```
Comment (KennethEnevoldsen) [inline docs/datasheets/danews2.0.md:19]: Newline here ?

File: docs/datasheets/danews2.0.md [hunk 3]
```diff
@@ -1,2 +1,55 @@
...
+An example from the dataset looks as follows.
+```yaml
+{
+    'id': 'e2a069b9',
+    'text': 'Slottet sænker vindebroen 2011-04-12T00:00:00Z NYB...', 'source': 'danews2.0',
```
Comment (KennethEnevoldsen) [inline docs/datasheets/danews2.0.md:24]: source on a newline?

File: docs/datasheets/danews2.0.md [hunk 4]
```diff
@@ -1,2 +1,55 @@
...
+    'added': '2024-05-16',
+    'created': '2011-04-12, 2012-04-11',
+    'metadata': {
+        'sub-source': 'LokalAvisen Nyborg',
+        'ArticleUrl': 'https://mediaresearchapi.infomedia.dk/api/v1/article?id=e2a069b9',
+        'Captions': [' Også i år skal der foretages arkæologiske udgravninger ved Nyborg Slot. Her fortæller Østfyns Museers Lars Ewald ( i midten med hat) forrige sommer om fundet af " den hemmelige tunnel." ARKIVFOTO: JØRGEN HANSEN'], 'Authors': [''],
```
Comment (KennethEnevoldsen) [inline docs/datasheets/danews2.0.md:30]: author on a newline

File: docs/datasheets/memo.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+        'title': 'Lykke-Peer',
+        'subtitle': None,
+        'volume': None,
+        'year': 1870,
+        'pages': '183',
+        'illustrations': 'n',
+        'typeface': 'gothic',
+        'publisher': 'Reitzel',
+        'price': '2,25',
+        'file_status': 'Modtaget fra KB 7.4.2022 DEL2 sending 5', 'notes': "OBS! PDF'en er ren tekst i antikva, men den fysiske bog formentlig fraktur. Det kalder på separate kolonner: pdf-typeface eller file-typeface og book-typeface. /PD",
```
Comment (KennethEnevoldsen) [inline docs/datasheets/memo.md:45]: notes on a newline

**Review by KennethEnevoldsen — COMMENTED** [review]
a few minor things

File: docs/datasheets/danews2.0.md (hunk 1 above)
Comment (TTTTao725) [reply docs/datasheets/danews2.0.md:47]: omg, I did it all wrong 😂 🤯

**Pull request merged**